chrome/ScreenSharing/README.md

```diff
@@ -88,7 +88,7 @@ Register you extension:
 
 ```javascript
 
-OT.registerScreenSharingExtension('chrome', 'YOUR-EXTENSION-ID');
+OT.registerScreenSharingExtension('chrome', 'YOUR-EXTENSION-ID', 2);
```

Comment:
I think this "version number" needs to be mentioned more prominently in the README file. After all, it is a breaking change. And the docs for `OT.registerScreenSharingExtension()` will have to describe the `version` parameter.

Please add this to the top of the README (before the "Customizing the extension for your website" heading):

This is version 2 of the extension. With version 2, the client can use the extension immediately after installing it using the inline installation method, without reloading the page. When calling the `OT.registerScreenSharingExtension()` method (see below), you must now pass in the version number, 2.